stats.go

```diff
@@ -112,3 +114,25 @@ func (s *Stats) Snapshot() *Stats {
 	})
 	return snap
 }
+
+func (s *Stats) String() string {
```

Comment:
I see. When I saw it output JSON-like data I was wondering. A simpler string output would have worked just as well, right?